CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ endif(MSVC)
 
 message("-- CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 
+if (${ENABLE_TESTS})
```

Comment:
As I don't understand CMake, maybe I'm making a stupid question. Anyway: Can this variable be called before the `option(ENABLE_TESTS …)` be declared?

I only saw the option in CMakeLists.txt of the subdirs ETL, synfig-core and synfig-studio.

Reply:
> Can this variable be called before the option(ENABLE_TESTS …) be declared?

Yes. You can set CMake variable from outside using command-line `-D` option. For example:
```
cmake -DENABLE_TESTS=1
```

`option` is used to add an option to the CMake GUI. 
This part also needs to be improved, but I want to back to it later. 

For example: do we need to add separate `tests` options for `core` and `studio`? Same for `docs` target, etc.


![cmake-gui](https://user-images.githubusercontent.com/5604544/116798545-b8088e80-ab1a-11eb-92b8-4bab582764cf.png)

Reply:
What I most miss in the CMake is features like `configure --help` and `configure --with-x` lol
I hope we get it some day.

Reply:
Is it easy to track all these variables for a later helper?

Reply:
You can use `cmake -L ../synfig` to see available variables:
```
Boost_INCLUDE_DIR:PATH=/usr/include
Boost_SYSTEM_LIBRARY_RELEASE:STRING=/usr/lib/x86_64-linux-gnu/libboost_system.so.1.71.0
CMAKE_BUILD_TYPE:STRING=Debug
CMAKE_INSTALL_PREFIX:PATH=/usr/local
DATA_PREFIX:PATH=/usr/local
ENABLE_TESTS:BOOL=0
GETTEXT_MSGFMT_EXECUTABLE:FILEPATH=/usr/bin/msgfmt
GETTEXT_MSGMERGE_EXECUTABLE:FILEPATH=/usr/bin/msgmerge
IMG_EXT:STRING=png
ImageMagick_EXECUTABLE_DIR:PATH=/usr/bin
ImageMagick_Magick++_ARCH_INCLUDE_DIR:PATH=/usr/include/x86_64-linux-gnu/ImageMagick-6
ImageMagick_Magick++_INCLUDE_DIR:PATH=/usr/include/ImageMagick-6
ImageMagick_Magick++_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libMagick++-6.Q16.so
MSGFMT_EXECUTABLE:FILEPATH=/usr/bin/msgfmt
WITHOUT_MAGICPP:BOOL=OFF
WITH_MAGICPP:BOOL=OFF
```